pkg/bpf/bpf.go

```diff
@@ -303,21 +303,34 @@ func (l *BpfLoader) setBpfProgOptions() {
 	nodeIP := getNodeIPAddress(node)
 	gateway := getNodePodSubGateway(node)
 
-	valueOfKmeshBpfConfig := KmeshBpfConfig{
-		// Write this map only when the kmesh daemon starts, so set bpfloglevel to the default value.
-		BpfLogLevel: constants.BPF_LOG_INFO,
-		NodeIP:      nodeIP,
-		PodGateway:  gateway,
-		// Use default values when bpf init.
-		// Updated when checking the startup parameters.
-		AuthzOffload:     uint32(0),
-		EnableMonitoring: uint32(1),
-	}
-
-	if err := UpdateKmeshConfigMap(l.kmeshConfig, &valueOfKmeshBpfConfig); err != nil {
-		log.Errorf("update kmeshConfig map failed: %v", err)
+	valueOfKmeshBpfConfig, err := GetKmeshConfigMap(l.kmeshConfig)
+	if err != nil {
+		log.Errorf("get kmeshConfig map failed: %v", err)
 		return
 	}
+
+	// Kmesh reboot updates only the nodeIP and pod sub gateway
```

Comment:
After restart, these two options do not change

Reply:
Done